### PR TITLE
Risco Local

### PR DESCRIPTION
**Pull request by OnFreund**
## Proposed change

Add documentation for the local version of the Risco integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/75874
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by OnFreund** [comment]
hmm, not sure why probot thinks this needs to go in `current`. It's a new feature.

File: source/_integrations/risco.markdown [hunk 1]
```diff
@@ -18,14 +18,17 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-This integration connects with Risco Alarms over [Risco Cloud](https://riscocloud.com/).
+This integration connects with Risco Alarms, in one of two ways:
+
+## Risco Cloud (recommended)
+
+The integration will connect with your alamr over [Risco Cloud](https://riscocloud.com/).
```
Comment (MartinHjelmare) [inline source/_integrations/risco.markdown]:
```suggestion
The integration will connect with your alarm over [Risco Cloud](https://riscocloud.com/).
```

File: source/_integrations/risco.markdown [hunk 2]
```diff
@@ -35,6 +38,17 @@ has the event timestamp as the state, and other event information in attributes.
 
 If you have multiple sites, only the first site will be used.
 
+## Local (advanced)
+
+The integration will connect locally to your system.
+No depency on the cloud, and instantenous updates, but is harder to set up.
```
Comment (MartinHjelmare) [inline source/_integrations/risco.markdown]:
```suggestion
No dependency on the cloud, and instantaneous updates, but is harder to set up.
```

File: source/_integrations/risco.markdown [hunk 3]
```diff
@@ -35,6 +38,17 @@ has the event timestamp as the state, and other event information in attributes.
...
+You will need the master code to your system, and with older models,
+you might need to either disconnect your system from the cloud, or setup a proxy that will allow you to connect both.
```
Comment (MartinHjelmare) [inline source/_integrations/risco.markdown]:
```suggestion
you might need to either disconnect your system from the cloud, or set up a proxy that will allow you to connect both locally and via the cloud.
```

File: source/_integrations/risco.markdown [hunk 4]
```diff
@@ -35,6 +38,17 @@ has the event timestamp as the state, and other event information in attributes.
...
+
+The loacl version of the integration does not support events, and the `arming` state.
```
Comment (MartinHjelmare) [inline source/_integrations/risco.markdown]:
```suggestion
The local version of the integration does not support events, and the `arming` state.
```

**Comment by OnFreund** [comment]
Oh wow, what happened to my spell checker :(

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**